st2common/st2common/router.py

```diff
@@ -178,6 +178,9 @@ def add_spec(self, spec, transforms):
                     for transform in transforms[filter]:
                         m.connect(None, re.sub(filter, transform, path), **connect_kw)
 
+                    module_name = endpoint['operationId'].split(':', 1)[0]
```

Comment:
Let's please also add a comment here and clarify why this is needed (basically what you wrote in the PR description).